src/input/CMakeLists.txt

```diff
@@ -50,15 +50,15 @@ else()
 endif()
 
 # Include MetavisionSDK
-find_package(MetavisionSDK COMPONENTS core driver)
-if (${MetavisionSDK_FOUND})
-    set(input_definitions ${input_definitions} WITH_METAVISION )
-    set(input_sources ${input_sources} prophesee.hpp prophesee.cpp)
-    set(input_libraries ${input_libraries} MetavisionSDK::core MetavisionSDK::driver)
-    set(input_include_directories ${input_include_directories})
-else()
-    message(STATUS "Prophesee dependencies (MetavisionSDK) not found, ignoring")
-endif()
+# find_package(MetavisionSDK COMPONENTS core driver)
```

Comment:
MetavisionSDK is only required for live use with the camera right? I can request hands on for the week after next (1st week of March) if we have something that theoretically should work.

Is adding task lists like this useful or is there a better way if we want to move this feature to a later PR?
- [ ] Check that it works with the open and closed source versions.

Reply:
Yes! And I was hoping we can do something like the `extras_require` setup for Metavision cameras. I don't know exactly how to handle that in practice, though. I opened an issue here: https://github.com/norse/aestream/issues/54